examples/micronaut-console-example-function/micronaut-console-example-function.gradle

```diff
@@ -16,20 +16,26 @@
  * limitations under the License.
  */
 // tag::import-runtime[]
+
 import com.amazonaws.services.lambda.model.InvocationType
 import com.amazonaws.services.lambda.model.Runtime
+import com.amazonaws.services.securitytoken.AWSSecurityTokenService
+
```

Comment:
it looks like the imports were reshuffled so they no longer match the asciidoc tags.